.github/workflows/main.yml

```diff
@@ -15,6 +15,8 @@ jobs:
       run: make
     - name: check
       run: make check
+    - name: cache
+      run: make cache
```

Comment:
The target should not be `cache`. Instead, it would be `make tests`, which triggers the automated testing for all available tests. At the moment, cache specific items are associated.